test/graphql/types/organizationConnectionList.spec.ts

```
@@ -0,0 +1,151 @@
+
+  it('should handle database errors gracefully', async () => {
+    // Arrange
+    const dbError = new Error('Database connection failed');
+    mockContext.drizzleClient.query.organizationsTable.findMany.mockRejectedValue(dbError);
+
+    // Act
+    const result = await executeOperation();
+
+    // Assert
+    expect(result.errors).toBeDefined();
+    expect(result.errors && result.errors[0]?.originalError).toBe(dbError);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)